src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/helpers.js

```diff
@@ -73,7 +73,7 @@ function displaySelectedMethod(type) {
   resetPaymentMethod();
 
   document.querySelector('button[value="submit-payment"]').disabled =
-    ['paypal', 'paywithgoogle', 'googlepay', 'amazonpay', 'applepay'].indexOf(
+    ['paypal', 'paywithgoogle', 'googlepay', 'amazonpay', 'applepay', 'klarna', 'klarna_account', 'klarna_paynow'].indexOf(
```

Comment:
I think you can only add klarna here since it uses indexOf, no need for klarna_account and klarna_paynow.